federatedscope/core/worker/client.py

```diff
@@ -254,17 +254,27 @@ def callback_funcs_for_evaluate(self, message: Message):
         self.state = message.state
         if message.content != None:
             self.trainer.update(message.content)
-        test_metrics = self.trainer.evaluate()
-        for key in test_metrics:
-            logging.info(
-                'Client #{:d}: (Evaluation (test set) at Round #{:d}) {:s} is {:.6f}'
-                .format(self.ID, self.state, key, test_metrics[key]))
-        val_metrics = self.trainer.validate()
-        for key in val_metrics:
-            logging.info(
-                'Client #{:d}: (Evaluation (val set) at Round #{:d}) {:s} is {:.6f}'
-                .format(self.ID, self.state, key, val_metrics[key]))
-        metrics = {**test_metrics, **val_metrics}
+        test_metrics, val_metrics, train_metrics = {}, {}, {}
+        if 'test' in self._cfg.eval.split:
```

Comment:
If @DavdGao @yxdyc  can unify the interfaces of trainer, we'd better use a for loop for these splits.